README.md

```diff
@@ -71,33 +77,28 @@ git_repository(
     tag = "0.3.1", # check for the latest tag when you install
 )
 
-load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories", "node_download_runtime", "yarn_download")
-
-node_download_runtime(
-    name = "nodejs",
-    version = "8.10.0",
-    packages = {
-        "darwin_amd64": ("node-v8.10.0-darwin-x64.tar.gz", "node-v8.10.0-darwin-x64", "7d77bd35bc781f02ba7383779da30bd529f21849b86f14d87e097497671b0271"),
-        "linux_amd64": ("node-v8.10.0-linux-x64.tar.xz", "node-v8.10.0-linux-x64", "92220638d661a43bd0fee2bf478cb283ead6524f231aabccf14c549ebc2bc338"),
-        "windows_amd64": ("node-v8.10.0-win-x64.zip", "node-v8.10.0-win-x64", "936ada36cb6f09a5565571e15eb8006e45c5a513529c19e21d070acf0e50321b"),
-    },
-    package_json = ["//:package.json"])
-
-yarn_download(
-    name = "yarn",
-    version = "1.5.1",
-    filename = "yarn-v1.5.1.tar.gz",
-    strip_prefix = "yarn-v1.5.1",
-    sha256 = "cd31657232cf48d57fdbff55f38bfa058d2fb4950450bd34af72dac796af4de1",
-    package_json = ["//:package.json"])
-
-# This rule will see the previously installed versions of NodeJS and Yarn from above and will skip attempting to set
-# them up.
+load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories")
+
 # NOTE: this rule does NOT install your npm dependencies into your node_modules folder.
 # You must still run the package manager to do this.
-node_repositories(package_json = ["//:package.json"])
+node_repositories(
+  node_version = "8.10.0",
+  yarn_version = "1.5.1",
+  node_repositories = {
+    "8.10.0-darwin_amd64": ("node-v8.10.0-darwin-x64.tar.gz", "node-v8.10.0-darwin-x64", "7d77bd35bc781f02ba7383779da30bd529f21849b86f14d87e097497671b0271"),
+    "8.10.0-linux_amd64": ("node-v8.10.0-linux-x64.tar.xz", "node-v8.10.0-linux-x64", "92220638d661a43bd0fee2bf478cb283ead6524f231aabccf14c549ebc2bc338"),
+    "8.10.0-windows_amd64": ("node-v8.10.0-win-x64.zip", "node-v8.10.0-win-x64", "936ada36cb6f09a5565571e15eb8006e45c5a513529c19e21d070acf0e50321b"),
+  },
+  yarn_repositories = {
+    "1.5.1": ("yarn-v1.5.1.tar.gz", "yarn-v1.5.1", "cd31657232cf48d57fdbff55f38bfa058d2fb4950450bd34af72dac796af4de1"),
+  },
+  node_urls = ["https://nodejs.org/dist/v{version}/{filename}"],
```

Comment:
at least for the "supported" versions we ought to offer a second mirror for downloads, from the mirror.bazel.build

Reply:
Good idea

Reply:
Doing the mirroring now